assets/Words.txt

```diff
@@ -1 +0,0 @@
-christianity, christian, islam, muslim, catholicism, catholic, hinduism, hindu, buddhism, buddhist, sikhism, sikh, seventh-day adventists, latter day saints, mormonism, mormon, judaism, jewish, jew, male, female, transgender, old, young, college, disabled, handicapped, single, married, couple, gay, straight, bisexual, no children, section 8, section eight, public assistance, lead paint, veteran, military
```

Comment:
I had utilized a csv file called "bad_words" (its in the slack channel) and thought it no longer necessary since I was putting these words within the bad_words csv for use later with the phrases table.